models/forms.rb

```diff
@@ -26,7 +26,7 @@ def message
 				"New organisation/account signup request from website",
 				"",
 				"From: #{person_name}",
-				"Email: #{person_email}",
+				"Email: #{person_email} (org manager: #{!!person_is_manager})",
```

Comment:
Is the double-bang necessary? We validate the model as containing booleans and the generated methods for boolean fields appear to do the double-bang thing anyway.

Reply:
I appreciate I wasn't at the kick-off but is it nicer to just have the text `(org manager)` next to the user that is going to be an OrgManager rather than having a possibly weird "(org manager: {true,false})` thing?

So, something like:

`#{"(org manager)" if person_is_manager}` or something?